website/server/libs/highlightMentions.js

```diff
@@ -111,6 +111,11 @@ function findTextBlocks (text) {
     const targetText = text.substr(index);
     const match = targetText.match(regex);
 
+    if (!match) {
+      // Should not happen, but insert to handle bugs gracefully
+      return;
```

Comment:
Ideally I would like to log an error here. If `console#log` is not the way to do that (Based on that there's a linter rule against it) what would be the best way to do that?

Reply:
You can use the logger interface like here https://github.com/HabitRPG/habitica/blob/develop/website/server/libs/pushNotifications.js#L149. It will log to the console in development and to an external service in production. Ideally the first argument should be an `Error` object while the second an object with extra optional data

Reply:
Awesome. Exactly what I needed. I don't expect to be able to hook up my laptop to the internet any time soon so perhaps merge this and I'll do the proper logging in a followup PR?

Reply:
Agreed, this way it'll go live with a deployment later today